frontend_new/src/components/Logo.tsx

```diff
@@ -0,0 +1,44 @@
+
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import CardMedia from '@material-ui/core/CardMedia';
+import logo from './frontend_new/src/components/logo.jpg';
+
+const useStyles = makeStyles({
+    root: {
+      minWidth: 275,
+    },
+    bullet: {
+      display: 'inline-block',
+      margin: '0 2px',
+      transform: 'scale(0.8)',
+    },
+    
+    title: {
+      fontSize: 14,
+    },
+    pos: {
+      marginBottom: 12,
+    },
+    
+  });
+
+export default function MediaCard() {
+    const classes = useStyles();
+    const bull = <span className={classes.bullet}>•</span>;
```

Comment:
**[Desired, Compatibility]** There is no guarantee that the bullet point character will actually show up in all browsers. Probably a good idea to use a unicode character with HTML encoding instead.